File: app/src/main/java/com/desuzed/clocknweather/util/TimeGetter.java
package com.desuzed.clocknweather.util;

import java.util.Calendar;
import java.util.GregorianCalendar;

public class TimeGetter {
    public Integer getHour (){
        return new GregorianCalendar().get(Calendar.HOUR);
    }
    public Integer getMinute (){
        return new GregorianCalendar().get(Calendar.MINUTE);
    }
    public Integer getSec(){
        return new GregorianCalendar().get(Calendar.SECOND);
    }
    public Integer getMSec (){
        return new GregorianCalendar().get(Calendar.MILLISECOND);
    }
}

File: app/src/main/java/com/desuzed/clocknweather/util/TimeGetter.kt
package com.desuzed.clocknweather.util

import java.util.*

class TimeGetter {
    val hour: Int
        get() = GregorianCalendar()[Calendar.HOUR]
    val minute: Int
        get() = GregorianCalendar()[Calendar.MINUTE]
    val sec: Int
        get() = GregorianCalendar()[Calendar.SECOND]
    val mSec: Int
        get() = GregorianCalendar()[Calendar.MILLISECOND]
}